### DISCUSSION TITLE
Mutiple issue fixes

### DISCUSSION THREAD

**Discussion by hsolbrig** [linkml/linkml#282] [Archived]
<hr /><em>This discussion was created from the release <a href='https://github.com/linkml/linkml/releases/tag/v1.0.3'>Mutiple issue fixes</a>.</em>

**Comment by cmungall** [comment]
did you mean to start a discussion @hsolbrig ?

**Comment by hsolbrig** [comment]
I did it just because I was curious what would happen.  New option...   

Now, how do we close it :-)
